docs/user/discover.asciidoc

```diff
@@ -195,6 +213,19 @@ image:images/visualize-from-discover.png[Visualization that opens from Discover
 
 . Save your visualization for use on a dashboard.
 
+Given your documents contain fields of type geo point, you can visualize them in **Maps**.
```

Comment:
This tutorial uses the sample weblog tutorial.  Does that contain a geo point we can use?

Otherwise, let's make this more of a generic statement.  

If your documents contain fields of type geo point, you can visualize them in Maps. From the Available fields list, click geo point field and then click Visualize.

Reply:
I think you mean our ecommerce demo data, so I've adapted here, also added the icon for this field type.

Reply:
<img width="808" alt="Bildschirmfoto 2021-03-04 um 15 03 15" src="https://user-images.githubusercontent.com/463851/109975640-22f43f80-7cfb-11eb-8d1c-71cb19b3bb7b.png">